custom_components/husqvarna_automower/translations/da.json

```diff
@@ -0,0 +1,63 @@
+{
+    "config": {
+        "step": {
+            "password": {
+                "description": "Hvis du har brug for hjælp til konfigurationen så kig her: https://github.com/Thomas55555/husqvarna_automower",
+                "data": {
+                    "api_key": "API Key (36 tegn lang)",
+                    "username": "Brugernavn",
+                    "password": "Password",
+                    "credentials": "Husk brugernavn og password (muliggører at denne komponent kan hente en ny token hvis den ikke længere kan fornyes)"
+                }
+            },
+            "user": {
+                "menu_options": {
+                    "password": "Log ind med API key, brugernavn og password",
+                    "oauth2": "Log ind med Authorization Code Grant (yaml nødvendig)"
+                }
+            },
+            "reauth_confirm": {
+                "description": "Husqvarna Automower integrationen har brug for at genautentificere ved Husqvarna",
+                "title": "Genautentificer integration"
+            }
+        },
+        "error": {
+            "auth": "Sørg for at Authentication API'en og Automower-Connect API'en er forbundet med din API Key: https://developer.husqvarnagroup.cloud/ og Brugernavn/Password/API Key er korrekt.",
+            "api_key": "Der er noget galt med din API Key. Forny den på: https://developer.husqvarnagroup.cloud/",
+            "mower": "Sørg for at Automower-Connect API'en er forbundet med din API Key: https://developer.husqvarnagroup.cloud/",
+            "unknown": "Der er sket en uforudset fejl"
+        },
+        "abort": {
+            "missing_configuration": "Komponenten er ikke konfigureret. Følg venligst dokumentationen.",
+            "reauth_successful": "Genautentificeringen var succesfuld",
+            "single_instance_allowed": "Kun én konfigurering af Husqvarna Automower er tilladt."
+        }
+    },
+    "options": {
+        "step": {
+            "init": {
+                "data": {
+                    "enable_camera": "Aktiver kort-kamera"
+                },
+                "description": "Aktiver kort-kamera",
+                "title": "Husqvarna Automower indstillinger"
+            },
+            "config": {
+                "data": {
+                    "gps_top_left": "GPS koordinater for det øverste venstre hjørne af billedet",
+                    "gps_bottom_right": "GPS koordinater for det nederste højre hjørne af billedet",
+                    "mower_img_path": "Filsti til billede af plæneklipper",
+                    "map_img_path": "Filsti til billede af kortet"
+
+                },
+                "description": "Kamera indstillinger",
+                "title": "Husqvarna Automower indstillinger"
+            }
+        }
+    },
+    "system_health": {
+        "info": {
+            "can_reach_server": "Kan nå Husqvarna Token server"
+        }
+    }
```

Comment:
Can you add a translation for this?
https://github.com/Thomas55555/husqvarna_automower/blob/f8b8a28f01a331455016dd9321cae2589ed23eba/custom_components/husqvarna_automower/translations/en.json#L42-L43
 I've just changed the strings.json and it's new now.

Reply:
Fixed in da7ce26